templates/Visitor/Association/_leaveAssociation_form.html.twig

```diff
@@ -0,0 +1,5 @@
+<form  method="post" action="{{ path('removeMemberAssociation', {'id': association.id, 'user': app.user.id}) }}" onsubmit="return confirm('Voulez vous vraiment quitter l\'entreprise ?');">
+    <input type="hidden" name="_id" value={{ app.user.id }}>
```

Comment:
pour quoi faire ? tu envois un id qui est recupérable par un $this->getUser() non ?